ci/run-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+EXTRA_FEATURE=""
+
+if [[ $1 != "" ]]
+then
+    EXTRA_FEATURE="--features $1"
+fi
+
+cargo test \
+    --features "test-programs/test_programs" \
+    $EXTRA_FEATURE \
```

Comment:
Could this perhaps be `"$@"` to just forward all arguments to `cargo test` (and placed at the end). That way the `--locked` could also be passed in only with CI to make this a bit more useful to run locally.

Reply:
Good idea, it's also more future-proof!